modules/docs/mdx/10.0-UPGRADE_GUIDE.mdx

```diff
@@ -13,6 +13,8 @@ any questions.
 	- [useBanner](#useBanner)
 - [Deprecations](#deprecations)
   - [InputIconContainer](#input-icon-container)
+  - [Table](#table)
+  - [TableRow](#table-row)
```

Comment:
You can probably just reference `Table` here and down below mention that all exported members or components are being deprecated

Reply:
Updated. Could you re-check?